specs/commandline/verify.md

```diff
@@ -168,3 +170,25 @@ An example of output messages for a successful verification:
 Warning:  Always verify the artifact using digest(@sha256:...) rather than a tag(:v1) because resolved digest may not point to the same signed artifact, as tags are mutable.
 Successfully verified signature for localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
 ```
+
+### [Experimental] Verify container images in OCI layout directory
+
+Users should configure trust policy properly before verifying artifacts in OCI layout directory. According to trust policy specification, `registryScopes` property of trust policy configuration determines which trust policy is applicable for the given artifact. For example, an image stored in a remote registry is referenced by "localhost:5000/net-monitor:v1". In order to verify the image, the value of `registryScopes` should contain "localhost:5000/net-monitor", which is the repository URL of the image. However, the reference to the image stored in OCI layout directory doesn't contain repository URL information. Users can set `registryScopes` to the URL that the image is supposed to be stored in the registry, and then use flag `--scope` for `notation verify` command to determine which trust policy is used for verification. Here is an example of trust policy configured for image `hello-world:v1`:
+
+```jsonc
+{
+    "name": "images stored in OCI layout",
+    "registryScopes": [ "localhost:5000/hello-world" ],
+    "signatureVerification": {
+        "level" : "strict"
+    },
+    "trustStores": [ "ca:hello-world" ],
+    "trustedIdentities": ["*"]
+}
+```
+
+To verify image `hello-world:v1`, user should set the environment variable `NOTATION_EXPERIMENTAL` and use flags `--oci-layout` and `--scope` together. for example:
+
+```shell
+NOTATION_EXPERIMENTAL=1 notation verify --oci-layout --scope "localhost:5000/hello-world" hello-world:v1
```

Comment:
Shouldn't `localhost:5000/hello-world" hello-world:v1` be disk path?

Reply:
@priteshbandi Currently the trust policy spec doesn't specify what will be the registryScope for a local content on filesystem, so the scope value here is an example, that it could be a URL that the local content will be stored in the registry later phase, if it is configured in trust policy. The disk absolute path could be very long, and relative path could be duplicated with others. Welcome any suggestions. Since it is experimental, we could iterate it gradually.